lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -142,6 +142,8 @@ class SqlWalker implements TreeWalker
      * Map of all components/classes that appear in the DQL query.
      *
      * @var array
+     *
+     * @psalm-var array<string, array{metadata: ClassMetadata, mixed}>
```

Comment:
I was going to merge, then saw this… Isn't that equivalent to
```suggestion
     * @psalm-var array<string, array{metadata: ClassMetadata}>
```
Or is there something I am missing?

Reply:
to be honest, I don't remember that part. I didn't use psalter and I don't see why I would have added that manually. It was probably a psalm suggestion, even if that doesn't make a lot of sense. I'll revert that